src/scss/_nav.scss

```diff
@@ -16,6 +16,7 @@
 
 	@include oGridRespondTo(M) {
 		position: relative;
+		left: 20px;
```

Comment:
That value is just the default from o-ft-icons, so allowing to change that would mean support to change the icon itself, and I don't think that's something we should support. I believe the design is that it should always be that size